config/operator/webhook/service.yaml

```diff
@@ -7,6 +7,6 @@ metadata:
 spec:
   ports:
     - port: 443
-      targetPort: 443
+      targetPort: 9443
```

Comment:
+100 for unprivileged port

Reply:
but I don't see a corresponding listen change:
a) there is some magic
b) we don't setup validation in e2e and that's why it passed

Reply:
I bumped controller-runtime version, and they changed default port, so a) :)